int32 symbol
int32 position
